kokoro/gcp_ubuntu/releases/release_utils.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+#
+# A Kokoro build script to run the extraction tool and integration tests
+
+# Code under repo is checked out to ${KOKORO_ARTIFACTS_DIR}/github.
+# The final directory name in this path is determined by the scm name specified
+# in the job configuration.
+
+# Display commands being run.
+# WARNING: please only enable 'set -x' if necessary for debugging, and be very
+#  careful if you handle credentials (e.g. from Keystore) with 'set -x':
+#  statements like "export VAR=$(cat /tmp/keystore/credentials)" will result in
+#  the credentials being printed in build logs.
+#  Additionally, recursive invocation with credentials as command-line
+#  parameters, will print the full command, with credentials, in the build logs.
+# set -x
+
+err() {
+  printf '%s\n' "[$(date +'%Y-%m-%dT%H:%M:%S%z')]: $*" >&2
+  exit 1
+}
+
+log() {
+  printf '%s\n' "[$(date +'%Y-%m-%dT%H:%M:%S%z')]: $*"
+}
+
+#######################################
+# makes HTTP requests and returns a status code
+# Globals:
+#   None
+# Arguments:
+#   A non optional HTTP header, a url
+# Outputs:
+#   A http status code
+#######################################
+http_get_error_code() {
+  http_header="$1"
+  http_url="$2"
+
+  http_response=$(curl -s -w "%{http_code}" -H ${http_header} ${http_url})
+
+  echo $(tail -n1 <<< "$http_response")
```

Comment:
Enclosing this in echo is superfluous, isn't it?

Reply:
Not sure if this is really critical. Please reopen if you think it is.